Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -169,9 +170,9 @@ extension Channel {
     /// - returns: `EventLoopFuture<Void>` that completes when the channel is ready.
     public func configureCommonHTTPServerPipeline(
         h2ConnectionChannelConfigurator: ((Channel) -> EventLoopFuture<Void>)? = nil,
-        _ configurator: @escaping (Channel) -> EventLoopFuture<Void>) -> EventLoopFuture<Void> {
+        _ configurator: @escaping (Channel) -> EventLoopFuture<Void>, targetWindowSize: Int32 = 65535) -> EventLoopFuture<Void> {
```

Comment:
This change falls foul of the new argument rule as well.

Reply:
This will definitely need documenting too. `targetWindowSize` should be the first argument as well so we can use trailing closure syntax.